### PR TITLE
[12.0][ADD] add fleet_vehicle_notebook

### PR DESCRIPTION
**Pull request by marcelsavegnago**

### CODE REVIEW

File: fleet_vehicle_notebook/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module extends the functionality of fleet management. It allows the registration of a vehicle's fuel capacity
```
Comment (ivantodorovich) [inline fleet_vehicle_notebook/readme/DESCRIPTION.rst]: Please update the description, also mentioning that this module doesn't really do anything by itself. 
it's just mean to be used by other modules

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: fleet_vehicle_notebook/readme/DESCRIPTION.rst (hunk 1 above)
Comment (marcelsavegnago) [reply fleet_vehicle_notebook/readme/DESCRIPTION.rst]: done

**Comment by marcelsavegnago** [comment]
ping @mamcode @mymage

**Review by mymage — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**